Python/ceval.c

```diff
@@ -747,24 +747,19 @@ _PyEval_InitRuntimeState(struct _ceval_runtime_state *ceval)
 #endif
 }
 
-int
-_PyEval_InitState(struct _ceval_state *ceval)
+void
```

Comment:
I don't get the point of this change.

Reply:
It's so I can call it in a void function.